.github/workflows/integration.yml

```diff
@@ -19,6 +19,10 @@ jobs:
       matrix:
         node-version: [10.x, 12.x, 14.x, 16.x, 18.x]
 
+    env:
```

Comment:
nit: you can move `OPENSEARCH_VERSION` to line 12 to avoid repetition and leave `SECURE_INTEGRATION` as is here.